src/Core/CMakeLists.txt

```diff
@@ -24,40 +24,33 @@ target_include_directories(ObEngineCore
 )
 
 if(OBE_USE_VCPKG)
+    find_package(fmt CONFIG REQUIRED)
```

Comment:
`find_package` in top-level CMakeLists.txt and then use the with `target_link_libraries` on targets ?